Fix `IterNextOutput::Return` not returning a value on PyPy.
